src/helpers/gcov.ts

```diff
@@ -15,5 +15,8 @@ export async function generateGcovCoverageFiles(projectRoot: string, include: st
     if (!files.length) {
         throw new Error('No gcov files found')
     }
-    return runExternalProgram('gcov', ['-pb', ...gcovArgs, ...files]);
+    if (gcovExecutable == 'gcov') {
+        gcovArgs.unshift('-pb')
```

Comment:
Where does `-pb` come from?

Reply:
Thanks, Tom. I thought `-pb` was being _removed_ from the array rather than added to it.

I clearly need to brush up on my JavaScript!